community/uthash/build

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh -e
 
-install -Dm644 src/* -t "$1/usr/include"
+install -d "$1/usr/include"
```

Comment:
`mkdir -p`

Reply:
This is useless now because `-D` flag automatically creates missing directories.